include/deal.II/fe/fe_values.h

```diff
@@ -3757,6 +3757,16 @@ class FEFaceValues : public FEFaceValuesBase<dim, spacedim>
                                             level_dof_access>> &cell,
          const unsigned int                                     face_no);
 
+  /**
+   * Alternative reinitialization function that takes an iterator to the
+   * current face instead of its number.
+   */
+  template <template <int, int> class DoFHandlerType, bool level_dof_access>
+  void
+  reinit(const TriaIterator<DoFCellAccessor<DoFHandlerType<dim, spacedim>,
+                                            level_dof_access>> &   cell,
```

Comment:
Why not use a `typename Triangulation<...>::cell_iterator` here, and a `face_iterator` for the next argument?

Reply:
I don't think the first one will work since we do need a `DoFCellAccessor` here in the subsequent `reinit` call. We can make the change for the face iterator, though.

Reply:
I misread. I really meant `DoFHandler<...>::cell_accessor. Why would that not work here?

Reply:
I don't think that it would work since we have a boolean for `level_dof_access` in this function: this is, I think, always false, for `DoFHandler::cell_accessor`.